frontend/src/modules/DatasetsBase/components/DatasetCreateWindow.js

```diff
@@ -17,27 +17,26 @@ import { isModuleEnabled, ModuleNames } from 'utils';
 
 export const DatasetCreateWindow = (props) => {
   const { onClose, open, ...other } = props;
-  const number_grid_items = isModuleEnabled(ModuleNames.S3_DATASETS) ? 2 : 0;
+  const number_grid_items =
```

Comment:
not for this PR: 
Following the pattern set by @SofiaSazonova at #1435 I think we should start thinking how to detangle UI from the `config.json`. Here we could have a query that returns all the enabled modules.

Reply:
Added in todo list in https://github.com/data-dot-all/dataall/issues/955